doc/release-notes.md

```diff
@@ -53,11 +53,15 @@ Removal of Priority Estimation
 
 - Estimation of "priority" needed for a transaction to be included within a target
   number of blocks has been removed.  The rpc calls are deprecated and will either
-  return -1 or 1e24 appropriately. The format for fee_estimates.dat has also
+  return -1 or 1e24 appropriately. The format for `fee_estimates.dat` has also
   changed to no longer save these priority estimates. It will automatically be
   converted to the new format which is not readable by prior versions of the
   software.
 
+- The concept of "priority" transactions is planned to be removed in the next
+  major version. To prepare for this, the default for the rate limit of priority
+  transactions (`-limitfreerelay`) has been set to `0` kB/minute.
```

Comment:
NACK. There is no plan to remove priority transactions, only "smart priority" attempting to use it. Don't confuse priority with gratis txns... they are different.